composer.json

```diff
@@ -8,5 +8,15 @@
     {
       "name": "Roman Nabiullin (nabiullin)"
     }
-  ]
+  ],
+  "require": {
+    "drupal/core": "^8.7.7 || ^9"
+  },
+  "extra": {
+    "drush": {
+      "services": {
+        "drush.services.yml": "^9"
```

Comment:
it should be `^9 || ^10` because drush 9 soon will be unsupported

Reply:
Yep, needs follow-up https://docs.drush.org/en/master/commands/#specifying-the-services-file